components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/user/core/common/AbstractUserStoreManager.java

```diff
@@ -682,26 +673,26 @@ private boolean authenticateInternal(String userName, Object credential, boolean
 
             int tenantId = abstractUserStoreManager.getTenantId();
 
-            try {
```

Comment:
Shall we remove the commented-out parts?

Reply:
we have to keep commented since tenant related code